cli/cli.go

```diff
@@ -736,6 +736,26 @@ func meshnetCommand(c *cmd) *cli.Command {
 						ArgsUsage:    MsgMeshnetPeerArgsUsage,
 						BashComplete: c.MeshPeerAutoComplete,
 					},
+					{
+						Name:  "nickname",
+						Usage: MsgMeshnetPeerNicknameUsage,
+						Subcommands: []*cli.Command{
+							{
+								Name:         "set",
+								Usage:        MsgMeshnetPeerSetNicknameUsage,
+								ArgsUsage:    MsgMeshnetPeerSetNicknameArgsUsage,
+								Action:       c.MeshPeerSetNickname,
+								BashComplete: c.MeshPeerNicknameAutoComplete,
+							},
+							{
+								Name:         "reset",
```

Comment:
Should it be changed to `remove` ?

Reply:
Or is it bound by API: nickname operations `set` and `reset` (ie. rename)?

Reply:
But I see API has single method to update machine peer. Why have to CLI commands to do the same? `set` and `reset` essentially do the same. Better to have `set` and `remove`, IMO. Otherwise how to remove nickname? Set it to empty?

Reply:
Or, does API operation mean that we `rename` peer and original peer name is gone? But I see that API gives back original peer name and nickname as well. So, it is not `peer rename` it is `set nickname`

Reply:
I've changed the commands names in the next PR https://github.com/NordSecurity/nordvpn-linux/pull/200/files#diff-ef44b92f1ba3916c39c98fc4a25b67da847cb2a07a5d1253e1978365338888a6R740 to `set`/`remove`
```
nordvpn mesh peer nickname set <ID|hostname|nickname> new-nickname
nordvpn mesh peer nickname remove <ID|hostname|nickname>
```
API exposes only one call, when calling with empty nickname means remove it.